src/book-search-result.js

```diff
@@ -0,0 +1,38 @@
+import { html, LitElement } from 'lit-element';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+
+export class BookSearchResult extends LitElement {
+  static get properties() {
+    return {
+      match: { type: Object },
+    };
+  }
+
+  constructor() {
+    super();
+
+    this.matchRegex = new RegExp('{{{(.+?)}}}', 'g');
+  }
+
+  createRenderRoot() {
+    return this;
+  }
```

Comment:
How come you're using the light dom?

Reply:
Taking advantage of the same style scope as the parent component.

Reply:
Hmm I don't think the parent should have that responsibility. The markup in here should be able to change independently of other components.